examples/Cargo.toml

```diff
@@ -0,0 +1,39 @@
+[package]
+name = "examples"
+version = "0.0.0"
+publish = false
+authors = ["Twilight Contributors"]
+edition = "2018"
+
+[dev-dependencies]
+ed25519-dalek = "1"
+futures-util = { default-features = false, version = "0.3" }
+hex = "0.4"
+hyper = { features = [
```

Comment:
we don't format our toml files like this, please put each dependency on one line even if it passes the wrap limit